File: java-lombok/src/main/java/it/gabrieletondi/telldontaskkata/domain/Order.java
package it.gabrieletondi.telldontaskkata.domain;

import lombok.Data;

import java.math.BigDecimal;
import java.util.List;

@Data
public class Order {
    private BigDecimal total;
    private String currency;
    private List<OrderItem> items;
    private BigDecimal tax;
    private OrderStatus status;
    private int id;
}

File: kotlin/src/main/java/it/gabrieletondi/telldontaskkata/domain/Order.kt
package it.gabrieletondi.telldontaskkata.domain

import java.math.BigDecimal

class Order {
    var total: BigDecimal? = null
    var currency: String? = null
    var items: List<OrderItem>? = null
    var tax: BigDecimal? = null
    var status: OrderStatus? = null
    var id = 0
}